one_fm/utils.py

```diff
@@ -782,96 +826,160 @@ def create_leave_allocation(employee, policy_detail, leave_type_details, from_da
         allocation.submit()
 
 def increase_daily_leave_balance():
-    # Get List of Leave Allocation for today (Leave Type - Is Paid Annual Leave)
+    '''
+        Function is used to Increase daily leave balance for Annual Leave Allocation
+        Triggered from daily scheduler event in hooks
+        Annual Leave Allocation:
+            It is the Leave Allocation for every employee with a leave type marked true for `Is Paid Annual Leave`
+    '''
+
+    # Get List of Leave Allocation for today of a Leave Type (having Is Paid Annual Leave marekd True)
+    allocation_list = get_paid_annual_leave_allocation_list()
```

Comment:
@pjamsheer Here you are not passing `date`, but in the function description, you are!

Reply:
since the definition having the argument value initialised

Reply:
👍